lib/Basic/LangOptions.cpp

```diff
@@ -564,11 +572,11 @@ std::pair<bool, bool> LangOptions::setTarget(llvm::Triple triple) {
   }
 
   // Set the "_pointerBitWidth" platform condition.
-  if (Target.isArch32Bit()) {
-    addPlatformConditionValue(PlatformConditionKind::PointerBitWidth, "_32");
-  } else if (Target.isArch64Bit()) {
-    addPlatformConditionValue(PlatformConditionKind::PointerBitWidth, "_64");
-  }
+    if (Target.isArch32Bit()) {
```

Comment:
unintentional whitespace change?

Reply:
yes, I thought I'd removed that, will fix

Reply:
I did a rebase that should fix it. Let me know if you're still seeing it.

Reply:
looks like its still there for some reason?

Reply:
This should (finally) be resolved. Apologies. My git gui was set to ignore whitespace so I was "blind" to the problem!